app/experimenter/nimbus-ui/src/components/PageResults/index.tsx

```diff
@@ -55,23 +57,40 @@ const PageResults: React.FunctionComponent<RouteComponentProps> = () => (
           </p>
           <h3 className="h6">Hypothesis</h3>
           <p>{experiment.hypothesis}</p>
-          <TableHighlights
-            primaryProbeSets={experiment.primaryProbeSets!}
-            results={analysis!}
-            {...{ experiment }}
-          />
+
+          {analysis?.overall ? (
+            <TableHighlights
+              primaryProbeSets={experiment.primaryProbeSets!}
+              results={analysis!}
+              {...{ experiment }}
+            />
+          ) : (
+            ""
+          )}
+
           <TableHighlightsOverview
             {...{ experiment }}
             results={analysis?.overall!}
           />
 
-          <h2 className="h5 mb-3" id="results-summary">
-            Results Summary
-          </h2>
-          <TableResults
-            primaryProbeSets={experiment.primaryProbeSets!}
-            results={analysis!}
-          />
+          <div id="results-summary">
+            <h2 className="h5 mb-3">Results Summary</h2>
```

Comment:
Will this display the "Results Summary" heading if we don't have either `overall` or `weekly` results, or will we always have at least one of those in this view?

Reply:
We would always have at least one of those in this view since there is a check above for `analysisUnavailable()`. At this point we know analysis is available which means at least one of `overall` or `weekly` results is available.